### PR TITLE
Add ScribbleMaps

### PR DESCRIPTION
**Pull request by Anshuman-Verma**
**http://scribblemaps.com/**

**Create custom google maps at ease, account needed to save these custom maps into your profile.**

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- [x] I have read the [Contribution guidelines](https://github.com/aviaryan/awesome-no-login-web-apps/blob/master/CONTRIBUTING.md) and I am confident that my PR is suitable. 
- [x] This pull request has a descriptive title. For example, `Add [App name]`, not `Update readme.md` or `Added new entries`.
- [x] The app I added is not a duplicate.

### CODE REVIEW

**Comment by aviaryan** [comment]
Maybe Drawing or Misc is a better category for it.

**Comment by Anshuman-Verma** [comment]
@aviaryan please review again.

File: README.md [hunk 1]
```diff
@@ -162,6 +163,7 @@ Powershell, Redis and lots others. Also has a good collection of IDEs for progra
 * [Typeracer](http://play.typeracer.com/) - Evaluate your typing speed while racing against others.
 
 
+
```
Comment (aviaryan) [inline README.md]: please remove this empty line.

Comment (Anshuman-Verma) [inline README.md]: @aviaryan fixed :+1:

**Pull request merged**